package.json

```
@@ -1,11 +1,16 @@
 {
   "name": "vscode-uri",
   "author": "Microsoft",
-  "version": "3.0.7",
+  "version": "3.0.8",
   "description": "The URI implementation that is used by VS Code and its extensions",
   "main": "./lib/umd/index.js",
   "typings": "./lib/umd/index",
-  "module": "./lib/esm/index.js",
+  "exports": {
+    "import": "./lib/esm/index.mjs",
+    "require": "./lib/umd/index.js",
+    "types": "./lib/umd/index.d.ts",
```

You may need to make the `types` export the first key to make it work for ESM.

```suggestion
    "types": "./lib/umd/index.d.ts",
    "import": "./lib/esm/index.mjs",
    "require": "./lib/umd/index.js",
```